pages/book/operators.mdx

```diff
@@ -27,7 +84,15 @@ Consider the following code:
 
 Even though this example may be simple, neglection of precedence rules can often lead to confusing situations with operators. The correct order of operations can be ensured by wrapping every operation in [parentheses](#parentheses), since parentheses have the highest precedence of all expressions and operators there is.
 
-## Parentheses, `()`
+That said, Tact's precedence levels are carefully crafted with predictability and convenience in mind, such that the following expression will always hold `true{:tact}`:
+
+```tact
+a / b * b + a % b == a; // true for any Int values of `a` and `b`,
```

Comment:
This is not about operator precedence level, it's a basic property of division. This should be moved to the corresponding section